test/sourcemap.coffee

```diff
@@ -133,50 +130,50 @@ unless process.version.slice(1, 3) is "12"
           equal column, 9 # Mapped source column
 
           resolve()
-        catch e
-          reject e
+        catch exception
+          reject exception
 
 test "don't change stack traces if another library has patched `Error.prepareStackTrace`", ->
   new Promise (resolve, reject) ->
-    proc = spawn "node", [
-      "--eval", """
+    proc = spawn 'node', [
```

Comment:
Should this one be `fork`?

Reply:
I don't think these can easily be switched to fork since it is using `--eval` and there is no `modulePath`. Though I could move these into `importing` and require them but I think that would obscure what is happening.

Reply:
Ah okay. Maybe just add a comment then that explains that we can't use `fork` with `--eval`?